src/main/java/de/redstoneworld/redworldinfo/RedSeedInfoCommand.java

```diff
@@ -0,0 +1,58 @@
+package de.redstoneworld.redworldinfo;
+
+import org.bukkit.ChatColor;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.HoverEvent;
+import net.md_5.bungee.api.chat.TextComponent;
+import net.md_5.bungee.api.chat.hover.content.Text;
+
+public class RedSeedInfoCommand implements CommandExecutor {
+	private final RedWorldInfo plugin;
+
+	public RedSeedInfoCommand(RedWorldInfo plugin) {
+		this.plugin = plugin;
+	}
+
+	public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
```

Comment:
I think for good measure (if something goes wrong) there should always be a permission check here.